### PR TITLE
Scene settings help button

### PR DESCRIPTION
**Pull request by AMZN-stankowi**
## What does this PR do?

Adds a help button to the scene manifest widget page, that links to documentation about the group associated with that page.

image

## How was this PR tested?

Manually tested so far. I'm investigating what our options are for capturing the URL open command in an automated test. That automated test will be a separate PR.

### CODE REVIEW

**Review by cgalvan — APPROVED** [review]

**Review by gadams3 — APPROVED** [review]

File: Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp [hunk 1]
```diff
@@ -138,6 +138,8 @@ namespace AZ::SceneAPI::SceneData
                         ->Attribute(AZ::Edit::Attributes::NameLabelOverride, "")
                         ->Attribute(AZ::Edit::Attributes::Max, 1)
                         ->Attribute(AZ::Edit::Attributes::CategoryStyle, "display divider")
+                        // There isn't a documentation page for default prefabs under the scene settings documentation category, yet.
```
Comment (junbo75) [inline Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp:141]: Nit: May worth cutting a GHI for that :)

**Review by junbo75 — APPROVED** [review]
May worth getting approval from UX. An alternative might be adding a tooltip to the "Add another mesh" button and include the link in the tooltip. The code change looks good.

File: Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp (hunk 1 above)
Comment (AMZN-stankowi) [reply Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp:141]: The prefab documentation https://www.o3de.org/docs/user-guide/interactivity/prefabs/ is a stub that links to a GHI https://github.com/o3de/o3de.org/issues/962 , which I thought was still open but is closed. I've asked for an update on the GHI and default prefab info.

**Comment by AMZN-stankowi** [comment]
> May worth getting approval from UX. An alternative might be adding a tooltip to the "Add another mesh" button and include the link in the tooltip. The code change looks good.

I talked to UX about this before posting the PR, there's where I got the icon to use from, too, so it's been approved by UX.

File: Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp (hunk 1 above)
Comment (micronAMZN) [reply Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroup.cpp:141]: We can get some useful info/links on this index page in the next few days.

**Pull request merged**